apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/style.scss

```diff
@@ -65,10 +65,10 @@
 		border-color: rgba( #425863, 0.4 ); // Gutenberg $dark-opacity-light-800
 	}
 
-	.editor-block-list__block:first-child & {
+	.block-editor-block-list__block:first-child & {
```

Comment:
This will require a new FSE release @Automattic/cylon

Reply:
If the CSS class is changing, it might be a good idea to keep the old one to handle the case where a user is still on an older Gutenberg version (unless this Gutenberg update has already been pushed to everyone on simple and atomic)

Reply:
I believe these refactors happened a while back in Gutenberg. This seems like the relevant PR: https://github.com/WordPress/gutenberg/pull/14420

Reply:
Ah yeah, removing legacy support would certainly do it. Thanks for adding the fix! By the way, how did you find that these broke?

Reply:
> how did you find that these broke 

Not sure I recall exactly, but probably something I spotted while looking into some of the e2e test failures. This in particular may have led me to it: https://github.com/Automattic/wp-calypso/pull/38857/files#diff-926c3450baa62272252cb16253634da9L27-R27

Reply:
We might still need to keep the old `.editor-` selectors in order to support old versions of Gutenberg. In any case, since these changes are unrelated to the E2E fixes, I'd move them to a separate PR.

Reply:
Ok, I can create a new PR with these changes.

Reply:
See my comment here please before I proceed: https://github.com/Automattic/wp-calypso/pull/38857/files#r369510379 I may be missing something.